tests/systemtests/protocolpool_test.go

```diff
@@ -0,0 +1,552 @@
+//go:build system_test
+
+package systemtests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tidwall/gjson"
+	"github.com/tidwall/sjson"
+
+	"cosmossdk.io/math"
+	"cosmossdk.io/systemtests"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+)
+
+const (
+	stakingModule      = "staking"
+	distributionModule = "distribution"
+	protocolPoolModule = "protocolpool"
+
+	stakingToken = "stake"
+)
+
+func modifyGovParams(t *testing.T) {
+	t.Helper()
+	// set up params so that we should just auto pass
+	systemtests.Sut.ModifyGenesisJSON(t,
+		func(genesis []byte) []byte {
+			state, err := sjson.Set(string(genesis), "app_state.gov.params.max_deposit_period", (1 * time.Second).String())
+			require.NoError(t, err)
+			return []byte(state)
+		},
+		func(genesis []byte) []byte {
+			state, err := sjson.Set(string(genesis), "app_state.gov.params.voting_period", (11 * time.Second).String())
+			require.NoError(t, err)
+			return []byte(state)
+		},
+		func(genesis []byte) []byte {
+			state, err := sjson.Set(string(genesis), "app_state.gov.params.veto_threshold", "0.000001")
+			require.NoError(t, err)
+			return []byte(state)
+		},
+		func(genesis []byte) []byte {
+			state, err := sjson.Set(string(genesis), "app_state.gov.params.threshold", "0.0000001")
+			require.NoError(t, err)
+			return []byte(state)
+		},
+		func(genesis []byte) []byte {
+			state, err := sjson.Set(string(genesis), "app_state.gov.params.quorum", "0.0000001")
+			require.NoError(t, err)
+			return []byte(state)
+		},
+	)
+}
+
+func TestQueryProtocolPool(t *testing.T) {
+	// Scenario:
+	// delegate tokens to validator
+	// check distribution
+
+	sut := systemtests.Sut
+	sut.ResetChain(t)
+
+	// set up gov params so we can pass props quickly
+	modifyGovParams(t)
+
+	cli := systemtests.NewCLIWrapper(t, sut, systemtests.Verbose)
+
+	// add genesis account with some tokens
+	account1Addr := cli.AddKey("account1")
+	sut.ModifyGenesisCLI(t,
+		[]string{"genesis", "add-genesis-account", account1Addr, "1000000000000stake"},
+	)
+
+	sut.StartChain(t)
+
+	// query validator address to delegate tokens
+	rsp := cli.CustomQuery("q", stakingModule, "validators")
+	valAddr := gjson.Get(rsp, "validators.#.operator_address").Array()[0].String()
+
+	// get validator address
+	valSigner := gjson.Get(cli.Keys("keys", "list"), "0.address").String()
+	require.NotEmpty(t, valSigner)
+
+	// stake tokens
+	rsp = cli.Run("tx", stakingModule, "delegate", valAddr, "10000000000stake", "--from="+account1Addr, "--fees=1stake")
+	systemtests.RequireTxSuccess(t, rsp)
+
+	beforeBalance := cli.QueryBalance(account1Addr, stakingToken)
+	assert.Equal(t, int64(989999999999), beforeBalance)
```

Comment:
i think it would be better to use integers where possible. i look at the assertion and think "is that right?" (not a lot of mental overhead, but would prefer to see math than magic numbers):

initialBalance := 1000000000000
amtStaked := 10000000000
fee := 1

assert balance is equal to initialBalance - amtStaked - fee